### DISCUSSION TITLE
old npm version in the documentation

### DISCUSSION THREAD

**Discussion by Tone866** [Koenkk/zigbee2mqtt#6748] [General]
I was rtfm and noticed this:

>\# Verify that the correct nodejs and npm (automatically installed with nodejs)
> \# version has been installed
> node --version  # Should output v12.X or v10.X
> npm --version  # Should output 6.X

npm's actual release is 7.x, so I was wondering if the docu is simply a bit outdated or if there where issues with the new npm version, so the 6.x is recommended.

**Comment by Koenkk** [comment]
Added npm 7 here. (depends on the installed node version)

**Reply by Tone866** [reply]
Ok, thank you!
